src/Symfony/Bridge/Doctrine/DataCollector/DoctrineDataCollector.php

```diff
@@ -117,8 +117,10 @@ private function sanitizeQueries($connectionName, $queries)
     private function sanitizeQuery($connectionName, $query)
     {
         $query['explainable'] = true;
-        $query['params'] = (array) $query['params'];
-        foreach ($query['params'] as $j => &$param) {
+        if (!is_array($query['params'])) {
+            $query['params'] = array($query['params']);
+        }
```

Comment:
I think `is_array($x) ? $x : array($x)` is more commonly used in the Symfony code base for these cases